styles.css

```diff
@@ -0,0 +1,535 @@
+    box-sizing: border-box;
+}
+
+a {
+    text-decoration: none;
+}
+
+nav ul {
+    list-style-type: none;
+}
+
+nav ul li a {
+    text-decoration: none;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+
+:root {
+    --purple: #5722CD;
+    --blue: #4B49E8;
+    --rose: #ED4385;
+    --black: #0C0C0C;
+    --magnolia: #F2F0FF;
+
+    --lightgrey: #F3F6F9;
+    --grey: #7A7A7A;
+    --white: #fdfeff;
+
+    --shadow: rgba(60, 60, 60, 0.4);
+}
+
+a {
+    color: var(--purple);
+}
+
+nav ul li a {
+    color: var(--magnolia);
+}
+
+button.pink {
+    color: var(--magnolia);
+    background-color: var(--rose);
+}
+
+header, section.contact {
+    color: var(--magnolia);
+    background-color: var(--blue);
+    background-image: radial-gradient(circle, var(--blue) 35%, var(--purple) 100%);
+}
+
+header h1, header h2 {
+    color: var(--white);
+}
+
+section.hello {
+    color: var(--black);
+    background-color: var(--white);
+}
+
+section.hello h1, section.hello h2 {
+    color: var(--purple);
+}
+
+section.work {
+    color: var(--black);
+    background-color: var(--lightgrey);
+}
+
+section.work h2 {
+    color: var(--black);
+}
+
+section.services {
+    color: var(--grey);
+}
+
+section.services article {
+    border-color: var(--purple);
+}
+
+section.services h2 {
+    color: var(--purple);
+}
+
+section.contact h1, section.contact h2 {
+    color: var(--white);
+}
+
+section.contact input, section.contact textarea {
+    background-color: var(--magnolia);
+}
+
+footer {
+    color: var(--magnolia);
+    background-color: var(--black);
+}
+
+footer section.copyright {
+    color: var(--magnolia);
+    background-color: var(--black);
+}
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* ------------------------------- 3.1 Fonts --------------------------------------- */
+
+body {
+    font-family: "Roboto", Arial, sans-serif;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-family: "Merriweather", Times, serif;
+}
+
+header h1, header h2 {
+    font-size: 2em;
+}
+
+header p {
+    font-size: 1.5em;
+}
+
+section.hello h1, section.hello h2 {
+    font-size: 1.25em;
+}
+
+section.hello p {
+    font-size: 1.1em;
+}
+
+section.work h1, section.work h2 {
+    font-size: 2em;
+}
+
+section.work p {
+    font-size: 1.3em;
+}
+
+section.contact h1, section.contact h2 {
+    font-size: 2em;
+}
+
+section.contact div > div > p {
+    font-size: 1.1em;
+    padding: 25px 0;
+}
+
+button {
+    font-size: 0.8em;
+    font-weight: 400;
+    letter-spacing: 0.05em;
+    text-align: center;
+}
+
+/* --------------- 3.1 Reusables ----------------------- */
+
+.nice-link {
+    font-size: 0.7em;
+    font-weight: 600;
+    letter-spacing: 0.15em;
+}
+
+.uppercase {
+    text-transform: uppercase;
+}
+
+.lowercase {
+    text-transform: lowercase;
+}
+
+button {
+    margin: 40px;
+    border: 0;
+    border-radius: 50vh;
+    padding: 20px 40px 20px 40px;
```

Comment:
Wist je dat als je `padding: 20px 40px` doet dat dat zich dan onderwater vertaald naar `padding: 20px 40px 20px 40px`? 😉

Reply:
Ja, weet ik en gebruik ik ook op andere plekken in mijn stylesheet. Maar niet heel consequent dus ... ;-)